### ISSUE TITLE
About pascal voc 2012 SBD training

### ISSUE THREAD

**Issue by Woolseyyy** [jfzhang95/pytorch-deeplab-xception#5]
I am training this model on pacal voc 2012 SBD dataset and I found that the mIoU improved slowly. 12 hours's training produced 0.41 mIoU in epoch 143. In the training process, I set learning rate as 1e-3, momentum as 0.90, batch size as 8 and weight decay as 5e-4. I  also use data augmentations: random rotate 10 degrees, random horizontally flip.
I modify the model according to tensorflow version. The details are shown in my forked repository.
Is this training speed normal?

**Comment by jfzhang95** [comment]
Based on my experience (train deeplab v2 on SBD, 100 epoch can reach a high mIoU in pascal voc 2012),
I think the training speed you reported maybe abnormal. 

Maybe you can try to convert tensorflow pretrained modified xception model into PyTorch version as pretrain model. Or maybe you can train deeplab v3+ from scratch. I do not have time 
to conduct these experiments currently, because I have to prepare for my GRE Test. T_T

So, if you successfully reproduce the deeplab v3+ results in PyTorch, welcome to submit your strategies~

**Comment by Meoooww** [comment]
> 根据我的经验（在 SBD 上训练 deeplab v2,100 epoch 可以在 pascal voc 2012 中达到很高的 mIoU），我认为您报告的训练速度可能异常。
> 
> 也许您可以尝试将 tensorflow 预训练修改后的 xception 模型转换为 PyTorch 版本作为预训练模型。或者，也许你可以从头开始训练deeplab v3+。我目前没有时间进行这些实验，因为我必须准备我的GRE考试T_T。
> 
> 所以，如果你在PyTorch中成功重现了deeplab v3+的结果，欢迎提交你的策略~

halo，请问一下大佬，我也有类似的问题，在其他的比如基于transformer上的分割模型，我的训练速度快得多，但是在您的这个代码里我要用很长的时间进行训练，我想问是否是您的代码缺少一些gpu或者并行处理上的优化？
